util/build_configs/cray-internal/chapel.modulefile.lua.template

```diff
@@ -0,0 +1,174 @@
+--[[
+
+    file chapel module
+
+    (C) Copyright 2020 Hewlett Packard Enterprise Development LP
+
+]]--
+
+
+-- reasons to keep module from continuing --
+
+
+-- no need to conflict with chapel Lmod has a one name rule
+conflict("cray-mpich2", "xt-mpich2")
+prereq("craype")
+
+--[[ a hugepages module is required for XC, not required for shasta at present 09/30/2020
+prereq_any(
+            "craype-hugepages2M",
+            "craype-hugepages4M",
+            "craype-hugepages8M",
+            "craype-hugepages16M",
+            "craype-hugepages32M",
+            "craype-hugepages64M",
+            "craype-hugepages128M",
+            "craype-hugepages256M",
+            "craype-hugepages512M",
+            "craype-hugepages1G",
+            "craype-hugepages2G"
```

Comment:
Seems unfortunate to have to maintain this list, but maybe that's the only option. Not a priority and nothing to hold up committing for, just stuck out.